doc/source/security/encrypt-traffic.rst

```diff
@@ -12,6 +12,52 @@ This includes traffic between client applications and cluster nodes,
    :local:
    :depth: 1
 
+There are two approaches to configure encryption: automatic and manual.
+
+.. _ssl-auto-conf:
+
+SSL Automatic Configuration
+===========================
+
+|PXC| includes the :variable:`pxc-encrypt-cluster-traffic` variable
+that enables automatic configuration of SSL encryption.
+By default, it is disabled, meaning that you need to configure SSL manually
+if you want to encrypt SST and IST traffic, and internal communication.
+
+This variable cannot be changed on the command line during runtime.
+To enable automatic configuration of SSL encryption,
+set ``pxc-encrypt-cluster-traffic=ON`` in the the ``[mysqld]`` section of
+the :file:`my.cnf` file, and restart the cluster.
+
+Automatic configuration of the SSL encryption needs key and certificate files.
+Creation of these files is covered by the :ref:`generate-keys-certs` section.
+Necessary key and certificate files are first searched at the ``ssl-ca``,
+``ssl-cert``, and ``ssl-key`` options under ``[mysqld]``.
+If these options are not set, it then looks in the data directory for
+:file:`ca.pem`, :file:`server-cert.pem`, and :file:`server-key.pem` files.
+
+.. note:: The ``[sst]`` section is not searched.
+
+If all three files are found, they are used to configure encryption for SST
+using XtraBackup (with ``encrypt=4`` mode, as explained in :ref:`xtrabackup`
+subsection).
+If any of the files are missing, a fatal error is generated.
+
+The following settings are applied (and overridden if necessary)::
+
+ [mysqld]
+ wsrep_provider_options=”socket.ssl_key=server-key.pem;socket.ssl_cert=server-cert.pem;socket.ssl_ca=ca.pem”
+
+ [sst]
+ encrypt=4
+ ssl-key=server-key.pem
+ ssl-ca=ca.pem
+ ssl-cert=server-cert.pem
+
+For :variable:`wsrep_provider_options`, only the mentioned options are affected
+(``socket.ssl_key``, ``socket,ssl_cert``, and ``socket.ssl_ca``),
+the rest are not modified.
+
 .. _generate-keys-certs:
 
 Generating Keys and Certificates
```

Comment:
"To encrypt |PXC| traffic, you must generate the following sets of files:" ....

It is not must to generate these key and certs. 

A story is like if the user
"Starting mysql-5.7, mysql will generate default key and cert and place them in data-directory. If user want to over-ride these certificates and place generate the new certificates then user can do so using the process laid below."

Reply:
In general please reduce the spacing of this "Generating Keys and Certificates" section. It seems like lot of things are spaced with 1.5 lines para or even more (especially commands + The following files are required:.... section)